ansible/vars/topo_t0-isolated-d128u128s2.yml

```diff
@@ -0,0 +1,3285 @@
+        ipv6: 2064:100::da/128
+      Ethernet1:
+        ipv4: 10.0.1.179/31
+        ipv6: fc00::366/126
+    bp_interfaces:
+      ipv4: 10.10.246.123/24
+      ipv6: fc0a::7b/64
+  ARISTA123T1:
+    properties:
+    - common
+    bgp:
+      asn: 64802
+      peers:
+        64601:
+          - 10.0.1.180
+          - fc00::369
+    interfaces:
+      Loopback0:
+        ipv4: 100.1.0.219/32
+        ipv6: 2064:100::db/128
+      Ethernet1:
+        ipv4: 10.0.1.181/31
+        ipv6: fc00::36a/126
+    bp_interfaces:
+      ipv4: 10.10.246.124/24
+      ipv6: fc0a::7c/64
+  ARISTA124T1:
+    properties:
+    - common
+    bgp:
+      asn: 64802
+      peers:
+        64601:
+          - 10.0.1.182
+          - fc00::36d
+    interfaces:
+      Loopback0:
+        ipv4: 100.1.0.220/32
+        ipv6: 2064:100::dc/128
+      Ethernet1:
+        ipv4: 10.0.1.183/31
+        ipv6: fc00::36e/126
+    bp_interfaces:
+      ipv4: 10.10.246.125/24
+      ipv6: fc0a::7d/64
+  ARISTA125T1:
+    properties:
+    - common
+    bgp:
+      asn: 64802
+      peers:
+        64601:
+          - 10.0.1.184
+          - fc00::371
+    interfaces:
+      Loopback0:
+        ipv4: 100.1.0.221/32
+        ipv6: 2064:100::dd/128
+      Ethernet1:
+        ipv4: 10.0.1.185/31
+        ipv6: fc00::372/126
+    bp_interfaces:
+      ipv4: 10.10.246.126/24
+      ipv6: fc0a::7e/64
+  ARISTA126T1:
+    properties:
+    - common
+    bgp:
+      asn: 64802
+      peers:
+        64601:
+          - 10.0.1.186
+          - fc00::375
+    interfaces:
+      Loopback0:
+        ipv4: 100.1.0.222/32
+        ipv6: 2064:100::de/128
+      Ethernet1:
+        ipv4: 10.0.1.187/31
+        ipv6: fc00::376/126
+    bp_interfaces:
+      ipv4: 10.10.246.127/24
+      ipv6: fc0a::7f/64
+  ARISTA127T1:
+    properties:
+    - common
+    bgp:
+      asn: 64802
+      peers:
+        64601:
+          - 10.0.1.188
+          - fc00::379
+    interfaces:
+      Loopback0:
+        ipv4: 100.1.0.223/32
+        ipv6: 2064:100::df/128
+      Ethernet1:
+        ipv4: 10.0.1.189/31
+        ipv6: fc00::37a/126
+    bp_interfaces:
+      ipv4: 10.10.246.128/24
+      ipv6: fc0a::80/64
+  ARISTA128T1:
+    properties:
+    - common
+    bgp:
+      asn: 64802
+      peers:
+        64601:
+          - 10.0.1.190
+          - fc00::37d
+    interfaces:
+      Loopback0:
+        ipv4: 100.1.0.224/32
+        ipv6: 2064:100::e0/128
+      Ethernet1:
+        ipv4: 10.0.1.191/31
+        ipv6: fc00::37e/126
+    bp_interfaces:
+      ipv4: 10.10.246.129/24
+      ipv6: fc0a::81/64
+  ARISTA01PT1:
+    properties:
+    - common
+    bgp:
+      asn: 64802
+      peers:
+        64601:
+          - 10.0.2.0
+          - fc00::401
+    interfaces:
+      Loopback0:
+        ipv4: 100.1.1.1/32
+        ipv6: 2064:100::101/128
+      Ethernet1:
+        ipv4: 10.0.2.1/31
+        ipv6: fc00::402/126
+    bp_interfaces:
+      ipv4: 10.10.246.130/24
+      ipv6: fc0a::82/64
+  ARISTA02PT1:
```

Comment:
hi Shawn, is it possible to change this to PT0? (not PT1)

Reply:
changed.